### ISSUE TITLE
metadata

### ISSUE THREAD

**Issue by andreasplesch** [andreasplesch/OCCToX3D#7]
Here is a fairly complete way to parse all the potential attributes of label:

http://git.dev.opencascade.org/gitweb/?p=occt.git;a=blob;f=src/XDEDRAW/XDEDRAW.cxx#l904

**Comment by tpaviot** [comment]
Porting to python should be straight forward, I don't see any major issue _a priori_.

**Comment by andreasplesch** [comment]
I agree. This is for the DFBrowser in draw and seems pretty comprehensive.
